jep/211/README.adoc

```diff
@@ -117,10 +117,22 @@ By default OpenJDK 10 and 11ea just print a warning once on the startup, there i
 === Scope of changes
 
 * Jenkins Core
-* Jenkins Docker Packaging
+* Docker Packaging for Debian
 * Plugins
-* jenkins.io
-* ci.jenkins.io
+* Jenkins website: jenkins.io
+* Jenkins CI instance: ci.jenkins.io and trusted-ci.jenkins.io
+* Maven Build flow
+* Test tools: Jenkins Test Harness, Acceptance Test Harness, Plugin Compatibility Tester
+
+Out of the scope for this JEP:
+
+* Packaging in subprojects: Jenkins X, Jenkins Evergreen, Jenkinsfile Runner, etc.
```

Comment:
> Packaging of subprojects:

Reply:
I meant to say that this packaging is happening in subprojects so that we exclude it for now. Not sure how to proper pass this wording